contracts/protocol/modules/LimitOrder.sol

```diff
@@ -63,6 +63,18 @@ abstract contract LimitOrderManager is ClaimableAmountManager {
                     _order.reduceLimitOrderId - 1
                 );
             }
+        } else if (_order.reduceQuantity != 0) {
+            if (_isReduce == 1) {
+                _orders[_orderIdx].reduceQuantity = partialFilled;
```

Comment:
if we cancel a reduce limit order, need to update reduce the quantity to partially filled amount